src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -337,9 +363,53 @@ class Container extends React.Component<Props, ComponentState> {
             entityNameVal: isPrebuilt ? getPrebuiltEntityName(obj.text) : prevState.isPrebuilt ? "" : prevState.entityNameVal,
         }))
     }
+
+    onDeleteEnum = (enumValue: CLM.EnumValue) => {
+        if (this.isEnumRequiredForActions(enumValue)) {
+            this.setState({
+                deleteEnumCheck: enumValue
+            })
+            return
+        }
+        else {
+            this.deleteEnum(enumValue)
+        }
+    }
+
+    deleteEnum(enumValue: CLM.EnumValue): void {
+        let index = this.state.enumValues.indexOf(enumValue)
```

Comment:
All these local variables can be `const`. They don't look to be modified below

Reply:
done.